src/main/java/com/iridium/iridiumskyblock/listeners/PlayerPortalListener.java

```diff
@@ -62,6 +62,8 @@ public void onPlayerPortal(PlayerPortalEvent event) {
                     event.setCancelled(true);
                     return;
                 }
+                location.setY(location.getY() + 1);
+                event.setCanCreatePortal(false);
```

Comment:
```suggestion
                if(XMaterial.supports(16)) event.setCanCreatePortal(false);
```
We could do this to try maintain 1.15- This wont fix the obsidian platform for 1.15, but atleast it will still be somewhat supported

Reply:
Origin code even doesn't support 1.16- ¯\_(ツ)_/¯
Because `java.lang.NoSuchFieldError: NETHER_WASTES`